compiler/src/lib.rs

```diff
@@ -166,7 +166,7 @@ fn compile<T: FieldElement>(
         log::info!("Wrote commits.bin.");
         if let Some(Backend::Halo2) = prove_with {
             use std::io::Write;
-            let proof = halo2::prove_ast(analyzed, constants, commits);
+            let proof = halo2::prove_ast(analyzed, constants, commits, None);
```

Comment:
Not critical for this PR but this should go through the `backend` crate as well

Reply:
Yea will do after